parsons/ngpvan/people.py

```diff
@@ -109,16 +100,18 @@ def upsert_person(self, first_name=None, last_name=None, date_of_birth=None, ema
         """
 
         return self._people_search(first_name, last_name, date_of_birth, email, phone,
-                                   street_number, street_name, zip, match_map, create=True)
+                                   street_number, street_name, zip, match_map, create=True,
+                                   validate=False)
 
     def _people_search(self, first_name=None, last_name=None, date_of_birth=None, email=None,
                        phone=None, street_number=None, street_name=None, zip=None, match_map=None,
-                       create=False):
+                       create=False, validate=True):
         # Internal method to hit the people find/create endpoints
 
         # Ensure that the minimum combination of fields were passed
-        self._valid_search(first_name, last_name, email, phone, date_of_birth, street_number,
-                           street_name, zip, match_map)
+        if validate:
+            self._valid_search(first_name, last_name, email, phone, date_of_birth, street_number,
+                               street_name, zip, match_map)
```

Comment:
I recognize this is was an attempt to quickly resolve one small feature request and one small bug, but there are larger issues with `_valid_search()` and it's possible it's worth resolving some of them now.
1. `street_number` and `street_name` are not processed anywhere in `_people_search()`, and I suspect they are relics of an older API version, because the Address object in the VAN documentation isn't broken out that way. I wonder if we shouldn't just get rid of them entirely until we can get an updated answer from VAN about how the validation logic works.
2. `_valid_search()` currently ignores `match_map` entirely. This might not be as urgent a bug to fix now that `upsert_person()` bypasses validation, but it still unnecessarily limits `find_person()`